app/controllers/v1/nominations_controller.rb

```diff
@@ -3,6 +3,10 @@ class V1::NominationsController < V1::BaseController
 
   MAPPINGS = {
     legislator_id:     'entry.353543474',
+    title:             'entry.1935590346',
+    name:              'entry.26710114',
+    state_abbrev:      'entry.1027714876',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.